Test failure on agent for TestManagersLargeRanges

Seems like there is a test on `agent`: `TestManagersLargeRanges` that is a bit flaky. Can reproduce locally.

```
<docker-logo-which-I-dont-have-because-no-emojis-on-my-terminal> test
?       github.com/docker/swarm-v2  [no test files]
--- FAIL: TestManagersLargeRanges (0.00s)
    managers_test.go:182: "one" not returned after several selection attempts, &{map[three:126 one:1 two:180] {0 0} [1.1 181.2 307.29999999999995] [one two three]}
FAIL
FAIL    github.com/docker/swarm-v2/agent    0.071s
```

/cc @LK4D4

--- a/agent/managers_test.go
+++ b/agent/managers_test.go
@@ -165,7 +165,7 @@ func TestManagersLargeRanges(t *testing.T) {
 	managers.Observe(addrs[2], managerWeightMax) // three bounces back!
 
 	seen := make(map[string]int)
-	for i := 0; i < len(addrs)*managerWeightMax; i++ {
+	for i := 0; i < len(addrs)*managerWeightMax*4; i++ {
 		next, err := managers.Select()
 		if err != nil {
 			t.Fatalf("error selecting manager: %v", err)

